xarray/core/utils.py

```diff
@@ -36,6 +37,20 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+def _maybe_cast_to_cftimeindex(index):
+    from ..coding.cftimeindex import CFTimeIndex
+    from ..coding.times import _require_standalone_cftime
+
+    if not OPTIONS['enable_cftimeindex']:
+        return index
+    else:
+        _require_standalone_cftime()
```

Comment:
Ideally, updating this code for v0.11 will be as simple as removing blocks following `if not OPTIONS['enable_cftimeindex']`.

This code wouldn't work in that case, because cftime isn't going to become a hard dependency of xarray.